packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -197,6 +206,30 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
         // Intentionally empty.
     }
 
+    // TODO: I don't like having so many return args but I also don't like
+    // wrapping them into a struct
```

Comment:
Agree neither are nice, but I think I prefer the struct...

Reply:
+1 - the struct is more readable imo. The tuple is a bit more efficient, but copying it back into memory wipes that efficiency away anyways.